NitroxServer/GameLogic/Players/PlayerData.cs

```diff
@@ -82,7 +82,7 @@ public class PersistedPlayerData
             public ushort Id { get; set; }
 
             [ProtoMember(5)]
-            public Vector3 SpawnPosition { get; set; }
+            public NitroxVector3 SpawnPosition { get; set; }
```

Comment:
If we were able to completely decouple the server then do we need to rename all of the types?  I think the types are only really used on the client when constructing objects for the server or marshalling data off of packets (so they can simply use the toDto())

Reply:
Not needed but the types are used in other projects where the distinction is useful to not have to use the `using` renaming.